README.md

```diff
@@ -120,13 +130,30 @@ The folders/files of the project `proj-dqry` (in combination with `object-detect
+  - [_Sentinel-2_](https://www.esa.int/Applications/Observing_the_Earth/Copernicus/Sentinel-2) (s2) image mosaics in false colour and 10 m spatial resolution are used for Brazil via the [BDC](https://data.inpe.br/bdc/web/en/home-page-2/).
+
+### Ground truth
+  - The MES labels of Switzerland come from the [_swissTLM3D_](https://www.swisstopo.admin.ch/fr/geodata/landscape/tlm3d.html) product. The file _mes_swisstlm3d_swissimage2020.shp_, used for training, has been reviewed and synchronised with the 2020 _SWISSIMAGE 10 cm_ mosaic. The file _mes_swisstlm3d_swissimage2020_landsat-fp-2020-08-11.shp_ has been adapted to the footprint of the Landsat image covering Switzerland on 11-08-2020.
+  - A global dataset of mining areas in the world has been compiled by [Maus et al. (2020)](https://www.nature.com/articles/s41597-020-00624-w) and can be downloaded [here](https://doi.pangaea.de/10.1594/PANGAEA.910894). The dataset has been adapted to our needs by synchronising them with image data and AoI (_mes_Maus2020_landsat-brazil-6m_2017-07_epsg4326.shp_ and _mes_Maus2020_s2-amazon-biome-3m_epsg4326.shp_).
+  - A dataset of artisanal gold mines detected in the Amazon, established by [Earth Genome](https://github.com/earthrise-media) since 2018, is available [here](https://github.com/earthrise-media/mining-detector). For the study we selected data from 2018 (_artisanal-mes_Earth-Genome_detections-2018.shp_) and 2022 (_artisanal-mes_Earth-Genome_detections-2022.shp_).
+
+### AoI
+  - The footprints of _SWISSIMAGE_ acquisition by year (_swissimage_footprint__*_.shp_) can be found [here](https://map.geo.admin.ch/?lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege,ch.astra.wanderland-sperrungen_umleitungen,ch.swisstopo.swissimage-product,ch.swisstopo.swissimage-product.metadata&layers_opacity=1,1,1,0.8,0.8,1,0.7&layers_visibility=false,false,false,false,false,true,true&layers_timestamp=18641231,,,,,2021,2021&time=2021). The shapefiles of _SWISSIMAGE_ acquisition footprint from 2015 to 2020 are provided in this repository.
+  - The polygon for the study area in the Brazilian Amazon is provided in this repository.
+
+### DEM and slope
+  The DEM and slope of Switzerland used in this project have been processed by Lukas Martinelli and can be downloaded [here](https://github.com/lukasmartinelli/swissdem).
+
+### Trained models
+  - The trained model used to produce the results with _SWISSIMAGE_ presented in the [documentation](https://github.com/swiss-territorial-data-lab/stdl-tech-website/tree/master/docs/PROJ-DQRY) and achieving a f1 score of 82% is available on request.
+  - The models trained on satellite images for the Swiss and Brazilian use cases presented in the [documentation](https://github.com/swiss-territorial-data-lab/stdl-tech-website/tree/master/docs/PROJ-DQRY) are available on request.
```

Comment:
I think it would be better to lead the reader to the formatted version of the doc on the tech website, rather than the github repo of the doc.